linkerd/announcer/curatorsd/src/main/scala/io/buoyant/linkerd/announcer/curatorsd/CuratorSDAnnouncerInitializer.scala

```diff
@@ -0,0 +1,19 @@
+package io.buoyant.linkerd.announcer.curatorsd
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.twitter.finagle.Path
+import io.buoyant.linkerd.{Announcer, AnnouncerConfig, AnnouncerInitializer}
+
+class CuratorSDAnnouncerInitializer extends AnnouncerInitializer {
+  override def configClass = classOf[CuratorSDConfig]
+  override def configId = "io.l5d.curatorsd"
+}
+
+case class CuratorSDConfig(zkConnectStr: String) extends AnnouncerConfig {
+
+  @JsonIgnore
+  override def defaultPrefix: Path = Path.read("/io.l5d.curatorsd")
```

Comment:
No, this is the linkerd config id for the announcer (explained in the javadoc).
The base path (chroot) can be specified in the zkConnectStr.